package.json

```diff
@@ -77,7 +84,12 @@
         "key": "alt+q",
         "when": "editorTextFocus"
       }
-    ]
+    ],
+    "configurationDefaults": {
```

Comment:
We should probably mention in README.md that installing will set the default formatter to ReflowMarkdown

Reply:
It will only do that if the user hasn’t explicitly set a formatter for Markdown in their settings.

So it will **not** override any existing settings. If you already have your preferred formatter set up, nothing changes. If you never used a formatter before, this will now be your default.